macOS/DuckDuckGo/MainWindow/MainViewController.swift

```diff
@@ -642,14 +642,14 @@ extension MainViewController {
              kVK_ANSI_R where flags == .command:
             guard view.window?.firstResponder is WebView else { return false }
             NSApp.menu?.performKeyEquivalent(with: event)
-            return true
+            return false
```

Comment:
@mallexxx, do you see any problems if events fall through to the web view in these particular cases?